src/main/webapp/app/overview/exercise-details/course-exercise-details.component.ts

```diff
@@ -173,6 +176,12 @@ export class CourseExerciseDetailsComponent implements OnInit, OnDestroy {
             this.exercise.studentParticipations.forEach(participation => {
                 this.participationWebsocketService.addParticipation(participation, this.exercise!);
             });
+            if (this.currentResult && this.currentResult.hasFeedback && !this.currentResult.successful) {
```

Comment:
You can here first check `this.currentResult` for both to not check it twice and then do `if - else if` inside

Reply:
Thanks for the hint. I changed that :)